### PR TITLE
LCOW: owner and network endpoints

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard <jhoward@microsoft.com>

Fixes libcontainerd to set the owner and network endpoints list. Stupid bug on my part. With this fix, ping works :smile:

```
PS E:\go\src\github.com\docker\docker> docker run -it --rm busybox ping www.google.com
PING www.google.com (172.217.3.196): 56 data bytes
64 bytes from 172.217.3.196: seq=0 ttl=51 time=1.804 ms
64 bytes from 172.217.3.196: seq=1 ttl=51 time=1.910 ms
64 bytes from 172.217.3.196: seq=2 ttl=51 time=1.984 ms
64 bytes from 172.217.3.196: seq=3 ttl=51 time=1.869 ms
64 bytes from 172.217.3.196: seq=4 ttl=51 time=2.066 ms
```

@johnstep @crosbymichael

### CODE REVIEW

File: libcontainerd/client_windows.go [hunk 1]
```diff
@@ -328,6 +329,18 @@ func (clnt *client) createLinux(containerID string, checkpoint string, checkpoin
...
 
+	for _, option := range options {
+		if n, ok := option.(*NetworkEndpointsOption); ok {
+			configuration.EndpointList = n.Endpoints
+			configuration.AllowUnqualifiedDNSQuery = n.AllowUnqualifiedDNSQuery
+			if n.DNSSearchList != nil {
+				configuration.DNSSearchList = strings.Join(n.DNSSearchList, ",")
+			}
+			configuration.NetworkSharedContainerName = n.NetworkSharedContainerID
+			continue
```
Comment (johnstep) [inline libcontainerd/client_windows.go]: If there is more than one NetworkEndpointsOption (will not happen with the current caller), is the idea that this loop continues, and uses the last one? In that case, the continue is not needed here. If there is only ever expected to be one, then `continue` should be replaced with `break`.

**Review by johnstep — APPROVED** [review]
LGTM when green.

**Comment by jhowardmsft** [comment]
@johnstep you are absolutely right. Updated continue-->break.

**Comment by jhowardmsft** [comment]
Green. @thaJeztah PTAL 😊

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Pull request merged**